sdk/python/foundation-models/system/finetune/chat-completion/chat-completion.ipynb

```diff
@@ -0,0 +1,731 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Chat Completion - Ultrachat-200k\n",
+    "\n",
+    "This sample shows how use `chat-completion` components from the `azureml` system registry to fine tune a model to complete a conversation between 2 people using ultrachat_200k dataset. We then deploy the fine tuned model to an online endpoint for real time inference.\n",
+    "\n",
+    "### Training data\n",
+    "We will use the [ultrachat_200k](https://huggingface.co/datasets/HuggingFaceH4/ultrachat_200k) dataset. This is a heavily filtered version of the UltraChat dataset and was used to train Zephyr-7B-β, a state of the art 7b chat model.\n",
+    "\n",
+    "The original datasets consists of 1.4M dialogues generated by ChatGPT and spanning a wide range of topics. To create UltraChat 200k, the following logic is applied:\n",
+    "\n",
+    "* Selection of a subset of data for faster supervised fine tuning.\n",
+    "* Truecasing of the dataset, as we observed around 5% of the data contained grammatical errors like \"Hello. how are you?\" instead of \"Hello. How are you?\"\n",
+    "* Removal of dialogues where the assistant replies with phrases like \"I do not have emotions\" or \"I don't have opinions\", even for fact-based prompts that don't involve either.\n",
+    "\n",
+    "**Dataset Structure**\n",
+    "\n",
+    "The dataset has four splits, suitable for:\n",
+    "* Supervised fine-tuning (sft).\n",
+    "* Generation ranking (gen) via techniques like rejection sampling or PPO.\n",
+    "The number of examples per split is shown as follows:\n",
+    "\n",
+    "| train_sft | test_sft | train_gen | test_gen |\n",
+    "| :- | :- | :- | :- |\n",
+    "| 207865 | 23110 | 256032 | 28304 |\n",
+    "\n",
+    "\n",
+    "The dataset is stored in parquet format with each entry using the following schema:\n",
```

Comment:
Refer to other notebooks on whether we are adding an example in the markdown section (or) not.

Reply:
https://github.com/Azure/azureml-examples/blob/main/sdk/python/foundation-models/system/finetune/translation/translation.ipynb